pageserver/src/billing_metrics.rs

```diff
@@ -186,7 +186,7 @@ pub async fn collect_metrics_task(
                 timeline_written_size,
             ));
 
-            let timeline_size = timeline.get_physical_size();
+            let timeline_size = timeline.get_resident_physical_size();
```

Comment:
Maybe we should rename the metric to `ResidentPhysicalSize` as well to be consistent? WDYT?

It is not used anywhere yet, so we can do it without breaking compatibility.

Reply:
Yes, will do that.